packages/utils/__tests__/ast-utils.test.ts

```diff
@@ -105,41 +105,41 @@ const a = { plugs: [] }
 	describe("createOrUpdatePluginByName", () => {
 		it("should create a new plugin without arguments", () => {
 			const ast = j("{ plugins: [] }");
-			ast.find(j.ArrayExpression).forEach(node => {
-				utils.createOrUpdatePluginByName(j, node, "Plugin");
+			ast.find(j.ArrayExpression).forEach((node) => {
```

Comment:
@ematipico I think we were agreeing on non-enclosed paran when single arg?

Reply:
Current `tslint` configuration is strict in those areas. `eslint` config is still relaxed.

Reply:
Can we make it as we agreed? Think Emanuelle has some comments there

Reply:
We will need to use "arrow-parens": [true, "ban-single-arg-parens"]" in tslint config 
more info: https://palantir.github.io/tslint/rules/arrow-parens/

Reply:
~`forEach((node) => {`~ `forEach((node: INode) => {`

@hemal7735 Do this for all occurrences of `node` and `path`. Check `ast-utils.ts` files for reference.